### PR TITLE
[smalltalk/en] Addition of an article for Smalltalk

### PR DESCRIPTION
**Pull request by jig08**

### CODE REVIEW

File: smalltalk.html.markdown [hunk 1]
```diff
@@ -0,0 +1,955 @@
...
+---
+
+- Smalltalk is an object-oriented, dynamically typed, reflective programming language.
+- Smalltalk was created as the language to underpin the "new world" of computing exemplified by "human–computer symbiosis."
+- It was designed and created in part for educational use, more so for constructionist learning, at the Learning Research Group (LRG) of Xerox PARC by Alan Kay, Dan Ingalls, Adele Goldberg, Ted Kaehler, Scott Wallace, and others during the 1970s.
+
+Feedback highly appreciated! Reach me at [@jigyasa_grover](https://twitter.com/jigyasa_grover) or send me an e-mail at `grover.jigyasa1@gmail.com`.
+
+
+##Allowable characters:
```
Comment (vendethiel) [inline smalltalk.html.markdown]: Allow**able**?

**Comment by vendethiel** [comment]
Thanks, but it looks less like a tutorial and more like a "list reference" of stuff?

**Comment by levibostian** [comment]
@vendethiel, do you have ideas on how to make it more of a reference? I am having troubles seeing what you mean so I am sure @jig08 does too. 

I am super excited to see this! I have been wanting to get into Smalltalk for a while now. Would love to get this merged in.

**Comment by vendethiel** [comment]
Well -- it's not a runnable piece of code is what I mean.

**Comment by jig08** [comment]
@levibostian , Thank You :smile:  Yes I observed `smalltalk` not being there hence thought of adding an article.

@vendethiel, I think I got your point.
(Maybe you could merge it for the time being for the sake of smalltalk's presence and create an issue for making it work like a executable piece of code. I will try to modify it to make it work that way and others could contribute too :smiley: )

**Comment by vendethiel** [comment]
Sure, that'd be fine by me :-)

**Comment by jig08** [comment]
Sure then :smiley:
Looking forward to `smalltalk`'s presence in [Learn X in Y minutes](https://github.com/adambard/learnxinyminutes-docs)

**Pull request merged**